README.md

```diff
@@ -15,56 +15,90 @@ Keyword Management System (KMS) is a application for maintaining keywords (scien
 
 To install the necessary components, run:
 
-```bash
-    npm install
+```
+npm install
 ```
 
 ### Usage
 
-In order to run KMS locally, you first need to setup a RDF database.
-
 #### Running Serverless Offline (API Gateway/Lambdas)
 
 In order to run serverless-offline, which is used for mimicking API Gateway to call lambda functions, run:
 
-```bash
-    npm run offline
+```
+RDF4J_SERVICE_URL=http://localhost:8080 npm run offline
 ```
 
 ## Local Testing
 
 To run the test suite, run:
 
-```bash
-    npm run test
 ```
-
-## Deploying RDF Database
+npm run test
+```
+## Setting up the RDF Database for local development
+In order to run KMS locally, you first need to setup a RDF database.
+### Prerequisites
+#### Set the RDFDB user name and password
+```
+export RDFDB_USER_NAME=[your rdfdb user name]
+export RDFDB_PASSWORD=[your rdfdb password]
+```
+### Building and Running the RDF Database
+#### Build the docker image
+```
+npm run rdf4j:build
+```
+#### Run the docker image
+```
+npm run rdf4j:start
+```
+#### Setup and load data into the RDF database
+```
+npm run rdf4j:setup
+```
+## Deploying RDF Database to AWS
 ### Prerequisites
 #### Copy your AWS credentials and set these up as env variables
-```export AWS_ACCESS_KEY_ID=[your access key id]
-export AWS_SECRET_ACCESS_KEY=[your access secret access key]
-export AWS_SESSION_TOKEN=[your session token]
+```
+export bamboo_STAGE_NAME=[sit|uat|prod]
+export bamboo_AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
```

Comment:
Split into 2 separate sections, one for deploying application, one for deploying the database, the bamboo stuff isn't needed for the RDF

Reply:
done